Documentation/Troubleshooting/performance-profiling.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Performance Profiling
+---
+
+## Collect perf data of a ceph process at runtime
+
+!!! warn
+    This is an advanced topic please be aware of the steps you're performing or reach out
+to the experts for further guidance.
```

Comment:
nit: I believe all lines of the warning need to be indented for it to be formatted properly
```suggestion
    to the experts for further guidance.
```